### ISSUE TITLE
claimTokens feature request

### ISSUE THREAD

**Issue by rstormsf** [OpenZeppelin/openzeppelin-solidity#507]
I'm curious if StandardToken should have this feature.
It's really helpful.

https://github.com/Giveth/minime/blob/master/contracts/MiniMeToken.sol#L535-L545
```
    function claimTokens(address _token) onlyController {
        if (_token == 0x0) {
            controller.transfer(this.balance);
            return;
        }

        MiniMeToken token = MiniMeToken(_token);
        uint balance = token.balanceOf(this);
        token.transfer(controller, balance);
        ClaimedTokens(_token, controller, balance);
    }
```
why: https://youtu.be/RL_e8fot29A?t=501 (Video where I explain this feature at 8:00)

**Comment by maraoz** [comment]
I wouldn't add it in `StandardToken`, but maybe a contract inheriting from that

**Comment by frangio** [comment]
The function is implemented in `CanReclaimToken`:
https://github.com/OpenZeppelin/zeppelin-solidity/blob/492ef4a419062043f217ad6a0ffb7c1bda14d42b/contracts/ownership/CanReclaimToken.sol#L20-L23

Additionally there is a proposal to provide a token with this feature included in #447.

**Comment by nventuro** [comment]
We removed `CanReclaimToken` in https://github.com/OpenZeppelin/openzeppelin-solidity/pull/1306, read there for rationale.
